sources/ghostdog/src/main.rs

```diff
@@ -1,6 +1,7 @@
 /*!
 ghostdog is a tool to manage ephemeral disks.
 It can be called as a udev helper program to identify ephemeral disks.
+It can also be called for EFA device detection which can be used for ExecCondition in systemd units.
```

Comment:
```suggestion
Also, it can be called to detect EFA devices
```

You don't have to explicitly call out the `ExecCondition` here.

Reply:
The above sentence mentioned `udev` for its use case, that is why I mentioned ExecCondition for this subcommand. In fact, it can only be used as `ExecCondition` since it will only emit exit code.